omod/src/main/webapp/resources/dhisconnector-runreports.js

```diff
@@ -139,6 +143,21 @@ function initializeYearlyPicker() {
     handleYearlyPeriodChange();
 }
 
+function initializeFinancialYearPicker(month) {
```

Comment:
I think we don't need two separate functionality forr yealy types. Use one instead

Reply:
So we can use the same for all yearly types(Yearly, FinancialOct, FinancialJuly, FinancialApril). 
Am i correct?

Reply:
yes